app/javascript/react/screens/App/Overview/components/AggregateCards/ActiveTransformationPlans/ActiveTransformationPlans.js

```diff
@@ -22,7 +23,18 @@ const ActiveTransformationPlans = ({ activePlans, allRequestsWithTasks, reloadCa
       const mostRecentRequest = requestsOfAssociatedPlan.length > 0 && getMostRecentRequest(requestsOfAssociatedPlan);
       return (
         mostRecentRequest &&
-        mostRecentRequest.miq_request_tasks.some(task => task.state === 'finished' && task.status === 'Error')
+        mostRecentRequest.miq_request_tasks.some(task => {
+          if (task.options && task.options.playbooks) {
+            const { playbooks } = task.options;
+            if (playbooks.pre && playbooks.pre.job_status === PLAN_JOB_STATUS.ERROR) {
```

Comment:
lines 29 to 34 could possibly be rewritten if yah wanted....

`return (playbooks.pre && playbooks.pre.job_status === PLAN_JOB_STATUS.ERROR) || (playbooks.post && playbooks.post.job_status === PLAN_JOB_STATUS.ERROR)